openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -710,7 +706,7 @@ def _objfunc(self, dv_dict):
             signal.signal(sigusr, self._signal_handler)
 
         try:
-            for name in self._indep_list:
+            for name in self._nl_dvs:
```

Comment:
I think co-pilot forgot to add `_nl_dvs` to the attributes list.

Reply:
fixed